lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/domain/dva/response/DvaResponse.java

```diff
@@ -8,35 +8,43 @@
 
 @ExcludeFromGeneratedCoverageReport
 public class DvaResponse {
-    private String correlationId;
+    // private String correlationId;
```

Comment:
Looking at the schema do we need all the fields in here?

Reply:
We might not need requestId, I've seen some conflicting example responses, some have it in some don't. My thinking was to leave there until we connect to real DVA, at which point we'll know more definitively what is and isn't needed